cloud/docker_client.go

```diff
@@ -414,19 +414,24 @@ func (c *dockerClientImpl) GetDockerLogs(ctx context.Context, containerID string
 }
 
 func (c *dockerClientImpl) GetDockerStatus(ctx context.Context, containerID string, parent *host.Host) (*ContainerStatus, error) {
-	if containerID == "" {
+	notRunningStatuses := []string{evergreen.HostUninitialized, evergreen.HostBuilding, evergreen.HostProvisioning, evergreen.HostStarting}
```

Comment:
There are a number of host status slices in globals.go (UphostStatus, DownHostStatus, etc.). Can you use one of those? If not, could you add your slice there?